js/page-specific/baltcogo-app/BaltCoGoReporterCtrl.js

```diff
@@ -374,6 +386,8 @@
 				}
 			});
 
+			var validRequiredElementsCount = requiredElements.filter('.ng-valid').length;
+
 			return requiredElementsCount === validRequiredElementsCount;
 		}
 
```

Comment:
would update the keycode conventions for 40 and 13. Could use a switch here, but not sure how you feel about those.  I found this https://www.safaribooksonline.com/library/view/high-performance-javascript/9781449382308/ch04s02.html which i don't think has convinced me either way for this situation.